pennylane/tensorbox/__init__.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```

Comment:
just  a thought, while the module name 'tensorbox' is logical, there may be a better one describing what it does? such as   `processing` or `maths` or `tensor`?

Reply:
I would rather not `maths` (as it sounds too useful/inviting for a user, who might explore `qml.maths`), or `tensor` (as we already have so many things called `tensor` in the codebase).

How about `qml.proc`? Short and sweet (`qml.proc.stack([x, y])`), but less inviting 😆

Reply:
what about `qml.processing`? if name is too long one can just import the module?

Reply:
I think we should avoid importing this module directly, since it will likely be used in so many places we will quickly end up in circular dependency hell 😆 

So we should always do
```python
import pennylane as qml
...
qml.processing.stack([x, y])
```

Reply:
Came here to say this, thanks @mariaschuld

Reply:
Done!